tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -329,13 +329,16 @@ def test_jstor_sets_config(self, js_config, jstor_service, pyramid_request):
 
 class TestAddCanvasSpeedgraderSettings:
     @pytest.mark.parametrize("group_set", (sentinel.group_set, None))
-    def test_it(self, js_config, pyramid_request, group_set):
+    def test_it(self, js_config, pyramid_request, course, assignment, group_set):
         pyramid_request.feature.return_value = False
         if group_set:
             pyramid_request.params["group_set"] = group_set
 
         js_config.add_canvas_speedgrader_settings(sentinel.document_url)
 
+        # Ensure `hypothesisClient` is added to config.
+        js_config.enable_lti_launch_mode(course, assignment)
```

Comment:
Not sure this is right, the tested method doesn't call `enable_lti_launch_mode`, we might be hiding some issue just to avoid the pylint disable (if that's the motivation for this change).

Reply:
The motivation for this change is to more directly test the output that will show up in the generated config. The method itself doesn't call `enable_lti_launch_mode`, but any consumer of `JSConfig` that wants to see it is going to need to call this, which I think is reasonable since this config is only used in LTI launches.